google-cloud-bigtable/lib/google/cloud/bigtable/instance_admin_client.rb

```diff
@@ -0,0 +1,424 @@
+# frozen_string_literal: true
+
+require "google/cloud/bigtable/admin/v2"
+require "google/bigtable/admin/v2/instance_pb"
+
+module Google
+  module Cloud
+    module Bigtable
+      # Protobuf aliases
+      Instance = Google::Bigtable::Admin::V2::Instance
+      Cluster = Google::Bigtable::Admin::V2::Cluster
+
+      class InstanceAdminClient
+        # Bigtable instance admin client for creating, configuring, and deleting
+        # instances and clusters
+
+        attr_reader :project_id, :options
+
+        # @param project_id [String]
+        # @param options [Hash]
+        def initialize project_id, options = {}
+          @project_id = project_id
+          @options = options
+        end
+
+        # Get instance information
+        #
+        # @param instance_id [String]
+        # @return [Google::Bigtable::Admin::V2::Instance]
+        # @example
+        #   require "google/cloud/bigtable"
+        #
+        #   client = Google::Cloud.bigtable(
```

Comment:
We have standardized on `client = Google::Cloud::Bigtable.new()` instead of `client = Google::Cloud.bigtable()` in code examples, except for the `#bigtable` and `.bigtable` methods.

We also don't usually show the `project_id` argument in most code examples, instead relying on GCE, environment variable, or configuration lookup to provide the value.